extensions/hp-extension/src/initialScale.ts

```diff
@@ -0,0 +1,6 @@
+export default function initialScaling(id,scaleFactor,viewport) {
+  console.log("Setting initial scaling on", viewport, scaleFactor);
```

Comment:
The code to set the initial scale is actually quite simple now, and does NOT require any changes to core OHIF beyond fixing the hanging protocol service.